addons/account/wizard/account_move_reversal.py

```diff
@@ -72,7 +72,10 @@ def default_get(self, fields):
             raise UserError(_("All selected moves for reversal must belong to the same company."))
 
         if any(move.state != "posted" for move in move_ids):
-            raise UserError(_('You can only reverse posted moves.'))
+            raise UserError(_(
+                'To reverse a journal entry, it has to be posted first.\n'
+                'That would be like asking for a credit note for something not invoiced: it’s fishy :)'
```

Comment:
Like for many other places: not necessarily an invoice

Reply:
Better English.. I guess, it's weird to use "it's fishy" in this context + it's another of those very culture specific expressions:

```suggestion
                'That would be like asking for a credit note for something that isn't invoiced: it’s fishy behavior :)'
```